api/src/pcapi/routes/backoffice/users/blueprint.py

```diff
@@ -230,3 +235,36 @@ def confirm_batch_suspend_users() -> utils.BackofficeResponse:
         flash(f"{len(users)} compte d'utilisateur a été suspendu", "success")
 
     return redirect(url_for("backoffice_web.fraud.list_blacklisted_domain_names"), code=303)
+
+
+@users_blueprint.route("/<int:user_id>/redirect_to_brevo", methods=["GET"])
```

Comment:
Pour que le style reste homogène, on utilise le « tiret du 6 » :
```suggestion
@users_blueprint.route("/<int:user_id>/redirect-to-brevo", methods=["GET"])
```